packaging/suse/velum.spec.in

```diff
@@ -36,14 +36,9 @@ Group:          System/Management
 %define velumdir /srv/velum
 
 Requires:       ruby >= 2.1
-%if 0%{?suse_version} >= 1210
-BuildRequires: systemd-rpm-macros
-%endif
 BuildRequires:  fdupes
 BuildRequires:  gcc-c++
 BuildRequires:  ruby-macros >= 5
-%{?systemd_requires}
-Provides:       velum = %{version}
```

Comment:
Is this intentional? If so, why are you removig the provides?

Reply:
Oh sorry, forgot to comment on this one. Basically I removed it because it causes a duplicate provides warning (see build logs of velum, between several warnings there is one related to duplicated provisioning). AFAIK this provides is implicit, you can see my [branch](https://build.opensuse.org/package/show/home:dcassany:branches:devel:CaaSP:Head:ControllerNode/velum) the resulting package still provides `velum = <version>`.

Reply:
Ok. Then please add a comment in the commit message. A part from that lgtm. I'll trust you will add the message in the commit, so I am approving it.